taskcluster/kinds/push-image/kind.yml

```diff
@@ -0,0 +1,59 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - docker-image
+
+transforms:
+    - scriptworker_taskgraph.transforms.push_image
+    - taskgraph.transforms.run:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    worker-type: images
+    description: "{name} image builder py{python_version}"
+    name: "{name}-python{python_version}"
+    worker:
+        taskcluster-proxy: true
+        docker-image: {in-tree: skopeo}
+        max-run-time: 1800
+        env:
+            VCS_HEAD_REPOSITORY: https://github.com/mozilla-releng/scriptworker-scripts
+            TASKCLUSTER_ROOT_URL: "$TASKCLUSTER_ROOT_URL"
+            # The following are set in the transform:
+            APP:
+            VCS_HEAD_REV:
+            DOCKER_REPO:
+            DOCKER_TAG:
+            DOCKER_ARCHIVE_TAG:
+    run-on-tasks-for: [github-push]
+    run-on-git-branches:
+        - dev
+        - production
+    run:
+        using: run-task
+        checkout: false
+        cache-dotcache: false
+        command:
+            - /usr/local/bin/push_image.sh
+    fetches:
+        image:
+            - artifact: image.tar.zst
+              extract: false
+
+tasks:
+    addonscript: {}
+    balrogscript: {}
+    beetmoverscript: {}
+    bouncerscript: {}
+    githubscript: {}
+    pushapkscript: {}
+    pushflatpakscript: {}
+    pushmsixscript: {}
+    shipitscript: {}
+    signingscript: {}
+    treescript: {}
```

Comment:
Might be nice to use `from_deps` transforms here, if for no other reason than to have one less place we need to list out all the *scripts. Not a big deal though.

Reply:
IMO it's worth keeping it explicit in this case, since it's a very limited list size-wise, with high impact when modified.